batch/indexer_Token_Holders.py

```diff
@@ -59,14 +63,16 @@ class BalanceBook:
         def __init__(self):
             self.pages = {}
 
-        def store(self, account_address: str, amount: int = 0):
+        def store(self, account_address: str, amount: int = 0, locked: int = 0):
             if account_address not in self.pages:
                 token_holder = TokenHolder()
                 token_holder.hold_balance = 0 + amount
                 token_holder.account_address = account_address
+                token_holder.locked_balance = 0 + locked
                 self.pages[account_address] = token_holder
             else:
                 self.pages[account_address].hold_balance += amount
+                self.pages[account_address].locked_balance += locked
```

Comment:
Added `locked_balance` field. Each method in processor calls this to accumulate locked amount.